receiver/adapter/accumulator/accumulator.go

```diff
@@ -200,7 +201,7 @@ func (o *otelAccumulator) modifyMetricAndConvertToOtelValue(m telegraf.Metric) (
 	}
 
 	if len(mMetric.Fields()) == 0 {
-		return nil, errEmptyAfterConvert
+		return nil, fmt.Errorf("empty metrics after converting fields: %w", errs)
```

Comment:
What does an example of a full error string look like in this case?

Reply:
As mentioned above, it'd be something like

```
empty metrics after converting fields: field (client): unsupported type: string; field (value): unsupported value: NaN
```

Reply:
Changed it to quote the fields, so it'd look like 

```
empty metrics after converting fields: field ("client"): unsupported type: string; field ("value"): unsupported value: NaN
```

Hopefully that makes it clearer that those are the fields.